### ISSUE TITLE
npm install fails

### ISSUE THREAD

**Issue by niklasp** [akella/balls#1]
at `node-sass@3.13.1 postinstall` : `Building: /usr/local/Cellar/node/14.14.0/bin/node /Users/n/code/balls/node_modules/node-gyp/bin/node-gyp.js` which node npm version are you using ?

**Comment by akella** [comment]
i think it was 6 or 8

**Comment by niklasp** [comment]
I don't know if you mind but you could add a `.nvmrc` file containing the version you use. It will make your repositories future safe for more people who follow your really nice work. Its a one liner, as described here: https://stackoverflow.com/questions/57110542/how-to-write-a-nvmrc-file-which-automatically-change-node-version#answer-57839539

**Comment by akella** [comment]
Good idea, i actually thought package.json had some automatical engine property, but i will add that to future ones. This one is old, to be honest, i dont even remember whats in this repo =) need to start it myself to see.
